addons/mrp_repair/models/repair.py

```diff
@@ -35,6 +35,15 @@ def action_explode(self):
         if line_vals_list:
             self.env['stock.move'].create(line_vals_list)
 
+    def _get_action_add_from_catalog_extra_context(self):
+        bom = self.env['mrp.bom']._bom_find(self.product_id, company_id=self.company_id.id)[self.product_id]
+        product_ids = [line.product_id.id for line in bom.bom_line_ids] if bom else []
+        return {
+            **super()._get_action_add_from_catalog_extra_context(),
+            'product_ids': product_ids,
```

Comment:
```suggestion
            'catalog_bom_product_ids': product_ids,
```
Seems too generic name. It could impact other flows if the context is not cleaned